examples/threadExample/build.gradle

```diff
@@ -30,4 +30,5 @@ tasks.preBuild {
 dependencies {
     compile 'com.android.support:appcompat-v7:21.0.+'
     compile files("../../realm/build/libs/realm-${version}.jar")
+    compile 'com.squareup.leakcanary:leakcanary-android:1.3.1'
```

Comment:
IMO it should be add to all apps, this is just hinting/enforcing a best practice